flang/include/flang/Lower/CharacterRuntime.h

```diff
@@ -66,11 +59,20 @@ void genTrim(Fortran::lower::FirOpBuilder &builder, mlir::Location loc,
              mlir::Value resultBox, mlir::Value stringBox);
 
 /// Generate call to verify runtime.
-/// This calls the descriptor based runtime call implementation of the verify
-/// intrinsic.
-void genVerify(Fortran::lower::FirOpBuilder &builder, mlir::Location loc,
+/// This calls the descriptor based runtime call implementation of the scan or
+/// verify intrinsics.
+void genScanVerify(Fortran::lower::FirOpBuilder &builder, mlir::Location loc,
```

Comment:
Please split this into two entry points: genScanDescriptor and genVerifyDescriptor to be consistent with genIndexDescriptor.